### ISSUE TITLE
Address security vulnerability: lodash

### ISSUE THREAD

**Issue by drewwmercer** [drewwmercer/developer-ocs#69]
Known high severity security vulnerability detected in lodash <4.17.5 defined in package-lock.json.
--
package-lock.json update suggested: lodash ~> 4.17.5.

**Comment by drewwmercer** [comment]
Updated 01-Oct-2019:
Remediation - Upgrade lodash to version 4.17.13 or later. 

For example:
```
"dependencies": {
  "lodash": ">=4.17.13"
}
```
or…
```
"devDependencies": {
  "lodash": ">=4.17.13"
}
```

Details: CVE-2019-10744
**critical severity**
Vulnerable versions: < 4.17.13
Patched version: 4.17.13
Affected versions of lodash are vulnerable to Prototype Pollution.
The function defaultsDeep could be tricked into adding or modifying properties of Object.prototype using a constructor payload.

**Comment by drewwmercer** [comment]
Working on this today.
